provider/aws/formation/rack.json

```diff
@@ -2932,7 +2932,11 @@
           "Statement": [
             {
               "Effect": "Allow",
-              "Action": [ "iam:*" ],
+              "Action": [
+                "iam:*Role",
```

Comment:
does `*` allowed in the begining?

Reply:
yes, we need access to get, create, delete, attach, detach roles
that's a shortcut.